app/static/src/app/types/responseTypes.ts

```diff
@@ -27,7 +27,7 @@ export interface StochasticConfig {
 }
 
 export interface Odin {
-    odin: () => unknown
+    new(...args : any[]): unknown
```

Comment:
Odin will get a bit more structure in later epics (e.g. will provide parameter), but these are tbd.